### PR TITLE
websocket read limits, raised on auth

### PR DESCRIPTION
**Pull request by chappjc**

### CODE REVIEW

File: server/comms/link.go [hunk 1]
```diff
@@ -11,6 +11,8 @@ import (
 	"decred.org/dcrdex/dex/ws"
 )
 
+const readLimitAuthorized = 65536
```
Comment (buck54321) [inline server/comms/link.go:14]: Use `math.MaxUint16`? OK to avoid an import though.

File: server/comms/link.go [hunk 2]
```diff
@@ -84,6 +90,15 @@ func (c *wsLink) IP() string {
 	return c.WSLink.IP()
 }
 
+// Authorized should be called from a request handler when the connection
+// becomes authorized. Unless it is run in a request handler synchronous with
+// other reads or prior to starting the link, it will be a data race with the
+// link's input loop. dex/ws.(*WsLink).inHandler odes does not run request
```
Comment (buck54321) [inline server/comms/link.go:96]: ~~odes~~

File: client/websocket/websocket.go [hunk 3]
```diff
@@ -109,6 +109,8 @@ func (s *Server) HandleConnect(ctx context.Context, w http.ResponseWriter, r *ht
 		return
 	}
 
+	// wsConn.SetReadLimit(65536) // if websocket reads need to be larger than ws.defaultReadLimit
+
```
Comment (buck54321) [inline client/websocket/websocket.go:113]: 4096 is probably good for unauthed connections.

**Review by buck54321 — APPROVED** [review]

File: server/comms/link.go (hunk 1 above)
Comment (chappjc) [reply server/comms/link.go:14]: This was actually quite random.  I don't want to imply there's some reason MaxUint16 is the best choice, unless the websocket reads really are limited to this.

**Pull request merged**